plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -1170,4 +1175,11 @@ static boolean isUnixSocketUri(URI uri) {
     private void createTarArchive(File tar, File... files) throws IOException {
         TarUtils.tarFiles(tar, 0, files);
     }
+
+    /**
+     * @return the container host ip address (from a docker container).
```

Comment:
See [tip](https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment) we gonna have the same rule.